package.json

```diff
@@ -29,7 +29,7 @@
     "type": "GitHub Sponsor",
     "url": "https://github.com/sponsors/wesleytodd"
   },
-  "license": "ICS",
+  "license": "ISC",
```

Comment:
```suggestion
  "license": "MIT",
```

Reply:
@wesleytodd I'm in favor, and it will bring it in line with the license of https://github.com/pkgjs/create